src/NServiceBus.Persistence.DynamoDB/Config/DynamoDBPersistenceConfig.cs

```diff
@@ -39,17 +41,19 @@ public static PersistenceExtensions<DynamoDBPersistence> TableName(this Persiste
             return persistenceExtensions;
         }
 
+        const string DisableTableCreationConfigKey = "NServiceBus.DynamoDB.CreateTables";
+        //TODO acceptance test
         /// <summary>
         /// Disables the tables creation.
         /// </summary>
         public static void DisableTablesCreation(this PersistenceExtensions<DynamoDBPersistence> persistenceExtensions)
         {
             Guard.AgainstNull(nameof(persistenceExtensions), persistenceExtensions);
-
-            var installerSettings = persistenceExtensions.GetSettings().GetOrCreate<InstallerSettings>();
-            installerSettings.Disabled = true;
+            persistenceExtensions.GetSettings().Set(DisableTableCreationConfigKey, true);
```

Comment:
Another option would be to directly use the `OutboxPersistenceConfiguration` and `SagaPersistenceConfiguration`?

Should we allow individual table types to be disabled?